src/main/scala/com/datastax/sparkstress/DataGenerator.scala

```diff
@@ -153,9 +149,12 @@ object RowGenerator {
     }
   }
 
-  def getPerfRowDataset(ss: SparkSession, seed: Long, numPartitions: Int, numTotalRows: Long, numTotalKeys: Long): org.apache.spark.sql.Dataset[PerfRowClass] = {
+  def getPerfRowDataFrame(ss: SparkSession, seed: Long, numPartitions: Int, numTotalRows: Long, numTotalKeys: Long): DataFrame = {
     import ss.implicits._
-    getPerfRowRdd(ss, seed, numPartitions, numTotalRows, numTotalKeys).toDS()
+    // There exists no encoder for Joda DateTimeObjects so let's build a tuple that the encoders can handle
+    getPerfRowRdd(ss, seed, numPartitions, numTotalRows, numTotalKeys).mapPartitions(
+      it => it.map( p => (p.store, new Timestamp(p.order_time.getMillis), p.order_number.toString, p.color, p.size, p.qty))
```

Comment:
Is it better to do the `toString` here or in the generator?

Reply:
i figured this way we skip doing "to String" when we don't have to, so RDDs don't have to go 
Object -> String -> Object
Datasets have to go
Object -> String -> SqlObject (which for UUIDs is string but for Timestamps is Long)